site/content/faq/what-is-the-default-directory-that-zap-uses.md

```diff
@@ -25,6 +25,6 @@ Note: ZAP's home is not the same as the user's home directory, which is also OS
 
 `~/Library/Application Support/ZAP`
 
-##  Weekly releases
+##  Weekly releases and development builds
 
-Note that for weekly releases `_D` is appended to the above directory. Development builds also use this directory.
+Note that for weekly releases `_D` is appended to the above directory. 
```

Comment:
```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the zap.log file will be saved in `~/.ZAP_D/`.
```

Reply:
Oh it didn't add it to the existing thread :cry: Silly GitHub interface.

Reply:
great

Reply:
Not sure about the zap.log reference, that's not mentioned anywhere else in this FAQ and we already have one for that.

Reply:
```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux `~/.ZAP_D/` will be used.
```

Is that better?